maximum_subarray/Cpp/max_subarray.cpp

```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int main(){
+	// Reading size of array and array.
+	cin >> n;
+	
+	for(int i=1;i<=n;i++)
+		cin >> arr[i];
+
+
+	for(int i=1;i<=n;i++){
+
```

Comment:
Redundant blank line at the start of a code block should be deleted.  [whitespace/blank_line] [2]

*Origin: CPPLintBear, Section: `all.cpplint`.*